src/Symfony/Component/Messenger/Stamp/TransportMessageIdStamp.php

```diff
@@ -18,7 +18,7 @@
  *
  * @experimental in 4.3
  */
-class TransportMessageIdStamp implements StampInterface
+final class TransportMessageIdStamp implements StampInterface
```

Comment:
not needed for sending but could be useful for debugging a message history (changing ID after retry)